src/assets/defaults.ts

```diff
@@ -901,3 +907,90 @@ export const defaultSensorDataloggerProfile: OverlayGrid = {
 export const defaultDisplayUnitPreferences = {
   distance: DistanceDisplayUnit.Meters,
 }
+
+export const defaultCockpitSettings = {
+  'cockpit-current-view-index': 0,
+  'cockpit-has-seen-tutorial': false,
+  'cockpit-auto-process-videos': true,
+  'cockpit-slide-events-enabled': true,
+  'cockpit-last-mission-name': null,
+  'cockpit-default-map-center': [-27.5935, -48.55854],
+  'cockpit-saved-profiles-v8': widgetProfiles,
+  'cockpit-slide-events-categories-required': {
+    'Arm': true,
+    'Disarm': true,
+    'Takeoff': true,
+    'Altitude Change': true,
+    'Land': true,
+    'Goto': true,
+  },
+  'cockpit-enabled-alert-levels': [
+    { level: 'info', enabled: false },
+    { level: 'success', enabled: true },
+    { level: 'error', enabled: true },
+    { level: 'warning', enabled: true },
+    { level: 'critical', enabled: true },
+  ],
+  'cockpit-mission-start-time': '2025-02-10T21:34:50.113Z',
```

Comment:
It's crucial that we don't change the default values from what they are today. This one should be `new Date()`, and not that random date.

The same happens for several others. I believe you copied your own settings instead of the real defaults.